packages/devtools_app/lib/src/memory/memory_allocation_table_data.dart

```diff
@@ -0,0 +1,159 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../table_data.dart';
+import 'memory_protocol.dart';
+
+const fieldWidthNumbers = 100.0;
```

Comment:
nit: `fieldWidthNumbers` --> `defaultNumberFieldWidth` or `numberFieldWidth`

Reply:
Changed to defaultNumberFieldWidth.